pkg/runtime/pubsub/bulksubscribe_events.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pubsub
+
+import (
+	"github.com/google/uuid"
+
+	"github.com/dapr/kit/logger"
+
+	contribPubsub "github.com/dapr/components-contrib/pubsub"
+)
+
+const (
+	Metadata = "metadata"
+	Entries  = "entries"
+)
+
+var bulkPSLogger = logger.NewLogger("bulk.subscribe")
+
+type BulkSubscribeMessageItem struct {
+	EntryID     string            `json:"entryID"`
+	Event       interface{}       `json:"event"`
+	Metadata    map[string]string `json:"metadata"`
+	ContentType string            `json:"contentType,omitempty"`
+}
+
+type BulkSubscribeEnvelope struct {
+	ID        string
```

Comment:
This is the ID field for this particular BulkSubscribe event. 
There are scenarios possible when entryIDs are sent like an index, 0, 1, 2, 3, 4 and so on...
While doing manual testing, I realised that for a user, it may become very tough to track that a particular event belonged to which bulk subscribe event. So, an ID is kept to be able to identify a particular bulk subscribe event.